### PR TITLE
[terra-progress-bar] A11Y updates

### PR DESCRIPTION
**Pull request by sugan2416**
### Summary

- A11y improvements for progress-bar
- Added visually hidden component under progress bar to provider context and status update of progress
- Updated test example
- Added translations

Closes #

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by JessieRandle** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please include a CHANGELOG entry for each changed package this PR. Looks like a CHANGELOG is missing for: 

 - terra-core-docs
 - terra-progress-bar</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ae781077fe8e88a43e2b2e49496ae61ee312da6a
</p>

**Comment by saket2403** [comment]
Can you update an example as well.

**Comment by sugan2416** [comment]
@saket2403 This test example was updated https://github.com/cerner/terra-core/blob/1369ca1fa31dc917949bcacdac79a2ae76ea3d02/packages/terra-core-docs/src/terra-dev-site/test/progress-bar/ProgressBarDefault.test.jsx to demonstrate use of aria-live for status and context. There are no other visual changes except label.

**Comment by saket2403** [comment]
> @saket2403 This test example was updated https://github.com/cerner/terra-core/blob/1369ca1fa31dc917949bcacdac79a2ae76ea3d02/packages/terra-core-docs/src/terra-dev-site/test/progress-bar/ProgressBarDefault.test.jsx to demonstrate use of aria-live for status and context. There are no other visual changes.

On the terra ui site user will not get to see the test page. So update an example under docs

**Comment by sugan2416** [comment]
> > @saket2403 This test example was updated https://github.com/cerner/terra-core/blob/1369ca1fa31dc917949bcacdac79a2ae76ea3d02/packages/terra-core-docs/src/terra-dev-site/test/progress-bar/ProgressBarDefault.test.jsx to demonstrate use of aria-live for status and context. There are no other visual changes.
> 
> On the terra ui site user will not get to see the test page. So update an example under docs

Added in https://github.com/cerner/terra-core/pull/3729/commits/939a686a704ec03d5cd822534de2938882ac8a7f

File: packages/terra-progress-bar/src/ProgressBar.jsx [hunk 1]
```diff
@@ -70,19 +71,23 @@ const ProgressBar = ({
   );
 
   const normalizedValue = (value / max) * 100;
+  const valText = valueText || `${normalizedValue}%`;
```
Comment (supreethmr) [inline packages/terra-progress-bar/src/ProgressBar.jsx:74]: percentage symbol `%` needs to be evaluated with translation team to ensure it conveys the same meaning on different locales

**Review by supreethmr — APPROVED** [review]
+1 with translation changes

**Comment by sugan2416** [comment]
> +1 with translation changes

Added translations in https://github.com/cerner/terra-core/pull/3729/commits/275881a93eb1ce6482513517be5401ca6d82fddb

**Pull request merged**